src/constants/colors/index.ts

```diff
@@ -1,186 +0,0 @@
-  statusRedBackground: '#F7EAE8',
-  statusRedBorder: '#F3DEDC',
-  statusGrey: DEFAULT,
-  statusGreyBackground: DEFAULT_BACKGROUND,
-  statusGreyBorder: DEFAULT_BORDER,
-
-  // Misc
-  black: '#000000',
-  white: '#ffffff',
-  yellow: '#eed153',
-  facebookBlue: '#305c99',
-} as const;
-
-export const legacyColors = {
-  charcoal: '#353a39',
-  grey1: '#999999', // numbers go from darkest to lightest
-  orange: '#fc7b65',
-  teal: '#d5e9e5',
-  yellowLight: '#f4f2b0',
-  purple80: '#5C5876',
-  purple60: '#858298',
-} as const;
-
-export const colorAliases = {
-  // Legacy
-  purple: brandColors.purple100,
-  purpleTint1: brandColors.purple85,
-  purpleTint2: brandColors.purple70,
-
-  // brand colors
-  primary: brandColors.purple100,
-  primaryTint1: brandColors.purple85,
-  primaryTint2: brandColors.purple70,
-  primaryTint3: brandColors.purple30,
-  primaryTint4: brandColors.purple15,
-  secondary: brandColors.lavender100,
-  tertiary: brandColors.purple4,
-
-  // general colors
-  success: brandColors.statusGreen,
-  successBackground: brandColors.statusGreenBackground,
-  successBorder: brandColors.statusGreenBorder,
-  info: brandColors.statusPurple,
-  infoBackground: brandColors.statusPurpleBackground,
-  infoBorder: brandColors.statusPurpleBorder,
-  error: brandColors.statusRed,
-  errorBackground: brandColors.statusRedBackground,
-  errorBorder: brandColors.statusRedBorder,
-  default: brandColors.purple70,
-  defaultBackground: brandColors.purple10,
-  defaultBorder: brandColors.purple15,
-  accent: brandColors.red,
-  disabled: brandColors.purple10,
-  failure: brandColors.red,
-  hover: brandColors.purple4,
-  warning: legacyColors.yellowLight,
-
-  // ui colors
-  background: brandColors.purple4,
-  backgroundDark: brandColors.purple4,
-  border: brandColors.purple10,
-  divider: brandColors.purple10,
-
-  // form colors
-  radioBorder: brandColors.purple30,
-  radioBorderSelected: brandColors.lavender100,
-
-  // typography
-  textMuted: brandColors.purple70,
-  textGhost: brandColors.purple30,
-  textDisabled: brandColors.purple30,
-
-  // overlay
-  overlay: 'rgba(45, 45, 48, 0.7)',
-  overlayHidden: 'rgba(45, 45, 48, 0)',
-  overlaySolid: 'rgba(45, 45, 48, 1)',
-  overlayTransition: 'rgba(45, 45, 48, 0.3)',
-} as const;
-
-export const postcardColors = {
-  postcardSocialBg: '#f2f2f2',
-  postcardWhite: brandColors.purple4,
-  postcardGreen: '#b8dca6',
-  postcardBlue: '#cae9f1',
-  postcardNude: '#e0c2b1',
-  postcardPeach: '#ffd3b0',
-  postcardOrange: legacyColors.orange,
-  postcardBlack: '#414141',
-  postcardGrey: '#bfc6cb',
-  postcardYellow: brandColors.yellow,
-} as const;
-
-// Dont change these keys -- will affect treatment page guide colors
-export const guideColors = {
-  sunscreenGuide: '#fcf4d5',
-  bigZitsGuide: '#f3d7be',
-  birthControlGuide: '#d0eae5',
-  oilySkinGuide: '#bdc5cb',
-  dietGuide: '#efd5ca',
-  drySkinGuide: '#f3f1a4',
-  makeupGuide: '#f7a9b7',
-  azelaicAcidGuide: '#9ddcc5',
```

Comment:
Is the plan to refer back to this diff when we encounter `LOCAL_COLORS` in usage? I'd also keep an eye out for `resources/assets/jsx/dashboard/components/guideCard/style.ts` which is dynamic and relies on all of these `guideColors`.

Reply:
yes, I am aware of the guides portion. I will work with v14 tommorrow or monday and cover all cases using to this PR